app/api/orders.py

```diff
@@ -194,6 +194,8 @@ def before_get(self, args, kwargs):
         :param kwargs:
         :return:
         """
+        if 'get_trashed=true' in str(request.query_string) and not has_access('is_admin'):
```

Comment:
That's a terrible way to check this

Reply:
@iamareebjamal I was actually looking for something which could indicate access for deleted objects. Any suggestion would be helpful

@mrsaicharan1 @shreyanshdwivedi please see

Reply:
@uds5501 I guess it's better to use `parse_qs` from `requests` library

Reply:
@mrsaicharan1 okay, taking a look at it

Reply:
Why can't you use kwargs?

Reply:
@iamareebjamal because the kwargs are only passing this : `{'event_id': 1}`

Reply:
Where does flask access get_trashed=true?

Reply:
It doesn't. The `get_trashed=true` is supposed to work on flask-rest-json api fork I think.
In this `before_get` hook, args is empty and the kwargs is being passed as I mentioned above